test/e2e/deviceplugins_suite_test.go

```diff
@@ -108,7 +108,7 @@ func setupFirstNode() []byte {
 	utils.Kubectl("node-feature-discovery", "apply", "-k", "deployments/nfd/overlays/node-feature-rules/kustomization.yaml")
 
 	if err = e2epod.WaitForPodsRunningReady(c, "node-feature-discovery", 2, 0,
-		100*time.Second, map[string]string{}); err != nil {
+		200*time.Second, map[string]string{}); err != nil {
```

Comment:
Using proxy/mirror didn't fully fix the e2e issues?

Reply:
I'm preparing for another change that makes the spr/gnr instances to run a bit slower.